Makefile

```diff
@@ -303,6 +303,11 @@ vcpus:
 vfio:
 	bash -f functional/vfio/run.sh -s false
 	bash -f functional/vfio/run.sh -s true
```

Comment:
please remove these two

Reply:
@devimc, updated, I am still debugging a failure to find the config file